inc/class-core-sitemaps.php

```diff
@@ -110,11 +111,11 @@ public function register_rewrites() {
 		add_rewrite_rule( '^wp-sitemap-index\.xsl$', 'index.php?sitemap-stylesheet=index', 'top' );
 
 		// Register routes for providers.
-		$providers = core_sitemaps_get_sitemaps();
-
-		foreach ( $providers as $provider ) {
-			add_rewrite_rule( $provider->route, $provider->rewrite_query(), 'top' );
-		}
+		add_rewrite_rule(
+			'^wp-sitemap-([A-z]+)-?([A-z]+)-?([0-9]+)?\.xml$',
```

Comment:
What permalink would return the users sitemap using this rewrite rule, since there is no sub-type for user objects?